ga4gh/datamodel/variants.py

```diff
@@ -47,6 +48,17 @@ def convertVCFGenotype(vcfGenotype, vcfPhaseset):
     return genotype, phaseset
 
 
+class CompoundVariantId(datamodel.CompoundId):
+    """
+    The compound id for a variant
+    """
+    fields = ['datasetId', 'vsId', 'chrom', 'pos', 'md5']
```

Comment:
OK, `variantSetId` is a special case, but `chrom` and `pos` should be `referenceName` and `start`.